### PR TITLE
feat(ui): Add Component column to TableCustom

### PR DESCRIPTION
**Pull request by bdauria**
This adds a column type for custom Component in TableCustom.
I will remove all KPI specific code from centreon-ui in a another step.

### CODE REVIEW

File: src/Table/TableCustom/index.js [hunk 1]
```diff
@@ -133,27 +134,187 @@ class TableCustom extends Component {
...
+          )}
+          {row[column.id] || ''}
+        </BodyTableCell>
+      ),
+      [TABLE_COLUMN_TYPES.toggler]: () => (
+        <BodyTableCell align="left">
+          {row[column.id] ? (
+            <Tooltip
+              label={labelEnableDisable}
+              onClick={(e) => {
```
Comment (kduret) [inline src/Table/TableCustom/index.js]: I'm not sure it's a good idea to enable/disable by clicking on tooltip

File: src/Table/TableCustom/index.js [hunk 2]
```diff
@@ -133,27 +134,187 @@ class TableCustom extends Component {
...
+                  e.preventDefault();
+                  e.stopPropagation();
+                  onDisable([row]);
+                }}
+              />
+            </Tooltip>
+          ) : (
+            <Tooltip
+              label={labelEnableDisable}
+              onClick={(e) => {
```
Comment (kduret) [inline src/Table/TableCustom/index.js]: I'm not sure it's a good idea to enable/disable by clicking on tooltip

File: src/Table/TableCustom/index.js [hunk 3]
```diff
@@ -133,27 +134,187 @@ class TableCustom extends Component {
...
+          </DefaultTooltip>
+        </BodyTableCell>
+      ),
+      [TABLE_COLUMN_TYPES.multicolumn]: () => (
+        <BodyTableCell key={column.id} align="left">
+          {column.columns.map((subColumn) => (
+            <>
+              {`${subColumn.label} ${row[subColumn.id]}`}
+              {subColumn.type === 'percentage' ? '%' : null}
+              {'   '}
```
Comment (kduret) [inline src/Table/TableCustom/index.js]: what is the purpose of those spaces ?

File: src/Table/TableCustom/index.js [hunk 4]
```diff
@@ -133,27 +134,187 @@ class TableCustom extends Component {
...
+                marginRight: -4,
+                position: 'absolute',
+                top: 3,
+                right: 0,
+              }}
+            >
+              <Box>
+                <Tooltip
+                  label={labelDelete}
+                  onClick={() => {
```
Comment (kduret) [inline src/Table/TableCustom/index.js]: I'm not sure it's a good idea to delete by clicking on tooltip

File: src/Table/TableCustom/index.js [hunk 5]
```diff
@@ -133,27 +134,187 @@ class TableCustom extends Component {
...
+                      e.stopPropagation();
+                      onDelete([row]);
+                    }}
+                  />
+                </Tooltip>
+              </Box>
+              <Box>
+                <Tooltip
+                  label={labelDuplicate}
+                  onClick={() => {
```
Comment (kduret) [inline src/Table/TableCustom/index.js]: I'm not sure it's a good idea to duplicate by clicking on tooltip

File: src/Table/TableCustom/index.js [hunk 6]
```diff
@@ -133,27 +134,187 @@ class TableCustom extends Component {
...
+      onDuplicate,
+      onDelete,
+      labelEnableDisable,
+      labelDuplicate,
+      labelDelete,
+    } = this.props;
+
+    const { hovered } = this.state;
+
+    const cellByColumnType = {
```
Comment (Thebarda) [inline src/Table/TableCustom/index.js]: I guess it is better to migrate this object to a class's function. That avoid to instantiate this object each time we are calling `getColumnCell`

**Pull request merged**